### ISSUE TITLE
Lack of screen reader support in `Select` and `Multi Select`

### ISSUE THREAD

**Issue by o-hook** [porsche-design-system/porsche-design-system#3952]
#### Description
If the `Select` and `Multi Select` are used the active options are not announced to screen reader users in VoiceOver/Safari.

### JS Framework

all

### Porsche Design System version

3.28.0

### On which device(s) you can see the bug?

all

### In which browsers you can see the bug?

VoiceOver/Safari

### Browser version

latest

### Reproduction Example

Storefront

### Steps to reproduce the behavior

1. Go to [Storefront](https://designsystem.porsche.com/v3/components/select/configurator/)
2. Click activate `filter` prop
3. Activate VoiceOver
4. Cycle through options --> nothing gets exposed to screen reader

### Expected behavior

The screen reader should announce the active descendant option element.
